src/foam/box/SessionClientBox.js

```diff
@@ -55,10 +56,8 @@ foam.CLASS({
           // that something occurred on the backend to destroy this user's
           // session. Therefore we reset the client state and ask them to log
           // in again.
-          if ( this.loginSuccess ) {
-            if ( this.ctrl )  this.ctrl.remove();
-            // Set loginSuccess to false so that if multiple requests are sent with no authentication, alert is called only once
-            this.loginSuccess = false;
+          if ( this.loginSuccess && ! this.subject.vacantMode ) {
+            if ( this.ctrl ) this.ctrl.remove();
```

Comment:
the 'this.loginSuccess = false;' was recently added https://github.com/kgrgreer/foam3/pull/769/files
how is this logic handled now?

Reply:
We need the the logic to call `alert` only once when user's session is destroyed (e.g., deleting user's session in backoffice). I tested on this branch so can confirm that without the logic, alert is called multiple times when session is destroyed.

Reply:
Makes sense to me - will be adding it back